docs/game_data/spel2.lua

```diff
@@ -73,36 +73,36 @@ function prinspect(...) end
 function messpect(...) end
 ---Adds a command that can be used in the console.
 ---@param name string
----@param cmd fun(): any
+---@param cmd function
 ---@return nil
 function register_console_command(name, cmd) end
 ---Returns unique id for the callback to be used in [clear_callback](#clear_callback). You can also return `false` from your function to clear the callback.
 ---Add per level callback function to be called every `frames` engine frames. Timer is paused on pause and cleared on level transition.
----@param cb fun(): any
```

Comment:
Maybe we should add the ones that are now missing in the comments?

Reply:
Agree